blocks/rich-text/index.js

```diff
@@ -371,6 +371,7 @@ export default class RichText extends Component {
 			plainText: this.pastedPlainText,
 			mode,
 			tagName: this.props.tagName,
+			allowIframes: this.context.canUserUseUnfilteredHTML,
```

Comment:
Are we ever expecting `allowScript`, `allowInput`, `allowStyle`?

Maybe we just pass the `canUserUseUnfilteredHTML` verbatim as the property name?